prebid-server/endpoints/openrtb2/adpod/video.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: page_v2
+sidebarType: 5
+title: Prebid Server | Endpoints | Video
+
+---
+
+# Prebid Server Video Endpoint
+
+This document describes the behavior of the Prebid Server video endpoint in detail.
+
```

Comment:
This is a long page, it should have a TOC at the top

Reply:
Added.